src/format/svg/SVGRenderer.hx

```diff
@@ -29,6 +29,7 @@ typedef ObjectFilter = String->GroupPath->Bool;
 
 class SVGRenderer
 {
+    public static inline var SQRT2:Float = 1.41421356237;

```

Comment:
Ah, I was just following this precedent: https://github.com/openfl/svg/blob/master/src/format/svg/SVGData.hx#L29-L30

But whatever, happy to use `Math.sqrt(2)`